agenta-backend/agenta_backend/routers/evaluation_router.py

```diff
@@ -60,6 +64,28 @@ async def create_evaluation(
         if app is None:
             raise HTTPException(status_code=404, detail="App not found")
 
+        if await check_if_ai_critique_exists_in_list_of_evlautors_configs(
+            payload.evaluators_configs
+        ):
+            # Check if lm_providers_keys is provided and not empty
+            if not payload.lm_providers_keys:
+                return JSONResponse(
+                    {"detail": "Missing Key"},
+                    status_code=400,
+                )
+
+            # Check if there is an OpenAI key and if it starts with 'sk-'
+            has_valid_open_ai_key = any(
+                key == LMProvidersEnum.open_ai and value.startswith("sk-")
```

Comment:
I don't think this is useful. Who does not say they change the prefix another day. Plus, it does not check for validity of the key (the real issue). I would remove this